arc.arc

```diff
@@ -2898,7 +2899,7 @@ Useful in higher-order functions, or to index into lists, strings, tables, etc."
             (= (savers* ',var) (fn (,gv) (,save ,gv ,file)))))))
 
 (mac diskvar (var file)
-  `(fromdisk ,var ,file nil readfile1 writefile))
```

Comment:
Hmm, now I'm wondering if we should also rename `readfile1` to `load-file` and so on.

But that piles on the incompatibilities. Others may object. What do you think, @hjek?